docs/deployment/managed-scanning.md

```diff
@@ -0,0 +1,194 @@
+1. Follow the prompts in GitHub to install the private app. Ensure that you grant access to the repositories you want to scan.
+
+:::caution Running into a 404?
+If you are brought to a GitHub 404 page, return to your [<i class="fas fa-external-link fa-xs"></i> GitHub Applications](https://github.com/settings/installations) page.
+:::
+
+### Completed installation
+
+A complete installation is displayed in the Source Code Manager entry as follows:
+
+![GitHub entry with public and private GitHub app connection](/img/zcs-code-access-enabled.png#md-width)
+_**Figure**. **Semgrep AppSec Platform > <i class="fa-solid fa-gear"></i> Settings > Source Code Managers** displaying a completed managed scanning set-up._
+
+You can also confirm a complete installation through your GitHub settings page, which should have two Semgrep apps:
+
+![GitHub settings page](/img/zcs-github-apps.png#bordered)
+_**Figure**. **GitHub > Settings > Applications** displaying both Semgrep apps. The private Semgrep app follows the convention **Semgrep Code - <span className="placeholder">YOUR_ORG_NAME</span>**_.
+
+## Add a repository
+
+1. In Semgrep AppSec Platform, click **<i class="fa-solid fa-folder-open"></i> Projects**.
+1. Click **Scan New Project > Semgrep Managed Scan**.
+1. Optional: If you can't find the repository you want to add, click **Can't find your project? > Sync projects**.
+1. Select the repositories you want to scan from the list.
+1. Click **Enable managed scanning**.
+
+- After enabling managed scanning, a full scan starts **immediately** on all the repositories you have added.
+- Once a repository has been added to Semgrep AppSec Platform, it becomes a **project**. A project in Semgrep AppSec Platform includes all the findings, history, and scan metadata of that repository.
+- Projects scanned through managed scanning are tagged with `autoscan`.
+
+### If the page doesn't display any repositories
+
+1. Ensure you have provided access to **both** the private and public Semgrep GitHub to the repositories you want to scan by following the steps in [Permissions and synchronicity](#permissions-and-synchronicity).
+1. In Semgrep AppSec Platform, click **<i class="fa-solid fa-folder-open"></i> Projects**.
+1. If the page doesn't display the repository you want to add, click **Can't find your project? > Sync projects**.
+1. If the page doesn't display any repositories, click **Sync projects**.
+1. Optional: Perform a hard refresh (<kbd>Ctrl</kbd>+<kbd>F5</kbd> or <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>R</kbd>).
+
+### How managed scanning detects repositories
+
+- Repositories with **existing** Semgrep CI jobs are **excluded** from the list. See the following section to convert your existing Semgrep CI jobs to managed scanning.
+- Repositories must be accessible to both the public Semgrep GitHub app and the private Semgrep GitHub app.
+
+### Convert or migrate an existing Semgrep CI job
```

Comment:
@s-santillan I had this wrong originally. It's actually simple to convert to Managed Scanning from CI scans:

1. Onboard the existing project to Semgrep Managed Scanning via the usual method. 
2. Remove the Semgrep.yaml file or cancel the CI job that runs scans in CI. 

That's all you need to do. Deleting a project will actually delete all of the findings so we don't want to encourage that.

Reply:
Does managed scanning pick up any specifics of the CI job settings?

Reply:
Any specifics: Managed scanning is like a vanilla semgrep.yml file. If you mean user-defined environment variables or additional steps in the yml file to divide a monorepo into submodules, it doesn't include any way to perform those steps. Hence the note on lockfile generation. I made a call out at the top and am wondering if we should call out again separately.